src/stylesheets/components/_service.scss

```diff
@@ -1,3 +1,46 @@
+$hover-text-color: #fff;
+$hover-bg-color: #00897b;
+
 .service {
-  
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  padding: 50px 10px;
+  background-color: v(bg-clr);
+
+  &:hover {
+    background-color: $hover-bg-color;
+    color: $hover-text-color;
+    box-shadow: 0px 5px 20px -10px black;
+    & .service__read-more {
+      border: 2px solid $hover-text-color;
+    }
```

Comment:
It's better to use css-variables for colors. Doing like this we may super easily switch colors with JS. It may be helpful if we want to add something like Dark Mode, and also it structures all colors in one place.